backend/testUtils/testSession.ts

```diff
@@ -1,10 +1,54 @@
-const mockTestSession = {
+import {
+  ResultRequestDTO,
+  TestSessionRequestDTO,
+  TestSessionResponseDTO,
+} from "../services/interfaces/testSessionService";
+
+export const testResult: ResultRequestDTO = {
+  student: "some-student-name",
+  score: 25,
+  answers: [10, 11],
+  breakdown: [false, true, true],
```

Comment:
wait - how can breakdown.length be > than answer.length lol

Reply:
Typo on my end - fixed now!

Reply:
i think the validation for this would be within `createTestResult`?

Reply:
yep exactly @carissa-tang